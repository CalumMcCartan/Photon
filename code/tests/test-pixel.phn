func int main()
{
    pixel a;
    a = _magenta;

    print(a[0]);
    print(a[1]);
    print(a[2]);
    print(a[3]);
    print(a.length);

    return 0;
}