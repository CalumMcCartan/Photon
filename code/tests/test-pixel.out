255
128
0
255